funcbench/README.md

```diff
@@ -28,8 +28,53 @@ The benchmark can be triggered by creating a comment in a PR which specifies a b
 /funcbench <branch/commit> <benchmark function regex>
 ```
 
+#### Setup
+
 The comment is handled by [comment-monitor](https://github.com/prometheus/test-infra/tree/master/tools/commentMonitor) and then the parsed arguments are handed over to funcbench(if using Github Actions) or to [prombench](https://github.com/prometheus/test-infra/tree/master/prombench) if using funcbench with GKE.
 
+- Create GitHub actions workflow file (see below) that is executed when an `repository_dispatch` event is on.
+- Read BRANCH / PR_NUMBER / REGEX from event payload into environment variables.
+
+#### Example GitHub action workflow file
+
+```yaml
+on: repository_dispatch
+name: Funcbench Workflow
+jobs:
+  run_funcbench:
+    name: Running funcbench
+    if: github.event.action == 'funcbench_start'
+    runs-on: ubuntu-latest
+    env:
+      AUTH_FILE: ${{ secrets.PROMBENCH_GKE_AUTH }}
+      CLUSTER_NAME: << cluster name >>
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PROJECT_ID: << project id >>
+      PR_NUMBER: ${{ github.event.client_payload.PR_NUMBER }}
+      TEST_INFRA_REPO: https://github.com/prometheus/test-infra.git # put your repo here if test your own test-infra
```

Comment:
Why do we need this link?

Reply:
I think this stayed after testing it with the prometheus-community repo for prombench, It can also be seen with the [prombench workflow file.](https://github.com/prometheus/prometheus/blob/386aea77744eac455c7430bd33cd0c736dba8a87/.github/workflows/prombench.yml#L28) keeping it here does make it easier to test though, as one will just need to copy and paste the workflow and just replace the variables.

Reply:
Should we remove it? as it's set to the same link in the Dockerfile?

Reply:
I use it for testing my fork. In real case, we don't need to set it.

Reply:
my question was really - what does this link specify? Why do we need to specify the test-infra repo? Does it download any files from there?

Reply:
Basically, it will clone the TEST_INFRA_REPO to WORKSPACE, then run `prombench` against TEST_INFRA_REPO, in this case, we can update the manifests without rebuilding the docker image.
As a reference, this `TEST_INFRA_REPO` was introduced in #266.

Reply:
yeah I remember now, don't think we should expose this in the workflow file as there is a very, very small chance someone would ever need to change this. Let's remove it from the example and leave it set to the default.